lib/Model.js

```diff
@@ -1270,10 +1270,22 @@ Model.batchPut = function(NewModel, items, options, next) {
   const schema = NewModel.$__.schema;
   const newModel$ = NewModel.$__;
 
+  let toDynamoOptions = {
+    updateTimestamps: false,
+    updateExpires: false
+  };
+
+  if (options.updateTimestamps === true) {
+    toDynamoOptions.updateTimestamps = true;
+  }
+  if (options.updateExpires === true) {
```

Comment:
Yes, I personally like this more (your first version). I was trying to copy the existing code style as much as possible. But since you suggested it yourself, I'll definitely change it. Not a fan of the second version, because it's slightly less telling and not as safe. So I'll go with the first one.

Reply:
@dolsem I'd like to refactor a lot of that code and fix up a lot of things to use more things like the versions I had above. It'd be a massive project but at some point I'd like to go through and clean up a lot of the Dynamoose code to make it more clean.

Not sure when that will happen tho. Until then I fully support new code using those type of clean coding practices.

Reply:
Okay, got you.